src/knapsack/knapsack.jl

```diff
@@ -1,4 +1,5 @@
 """
+    zero_one_pack!(capacity::N, weights::V, values::V, dp::V) where {N <: Number,V <: AbstractVector}
```

Comment:
Should this have an indentation or not (same in other parts of the code)?

Reply:
It should have the indentation, as you can see in the manual.

https://juliadocs.github.io/Documenter.jl/stable/man/guide/#Adding-Some-Docstrings